libcontainer/system/syscall_linux_64.go

```diff
@@ -1,6 +1,6 @@
-//go:build linux && (arm64 || amd64 || mips || mipsle || mips64 || mips64le || ppc || ppc64 || ppc64le || riscv64 || s390x)
+//go:build linux && (arm64 || amd64 || loong64 || mips || mipsle || mips64 || mips64le || ppc || ppc64 || ppc64le || riscv64 || s390x)
 // +build linux
-// +build arm64 amd64 mips mipsle mips64 mips64le ppc ppc64 ppc64le riscv64 s390x
+// +build arm64 amd64 loong64 mips mipsle mips64 mips64le ppc ppc64 ppc64le riscv64 s390x
 
```

Comment:
seccomp needs to be updated too

https://github.com/opencontainers/runc/blob/a2f27f05e86326f27134e79370f431c2c098348d/libcontainer/seccomp/config.go#L74
https://github.com/opencontainers/runc/blob/a2f27f05e86326f27134e79370f431c2c098348d/libcontainer/seccomp/patchbpf/enosys_linux.go#L82

Reply:
...and libseccomp-golang needs to be updated as well (see https://github.com/seccomp/libseccomp-golang/pull/106)

Reply:
> ...and libseccomp-golang needs to be updated as well (see [seccomp/libseccomp-golang#106](https://github.com/seccomp/libseccomp-golang/pull/106))

Sorry, I didn't know that you had submitted a PR before submitting the PR (https://github.com/seccomp/libseccomp-golang/pull/107), but there are some differences in the submitted code, please review

Reply:
> seccomp needs to be updated too
> 
> https://github.com/opencontainers/runc/blob/a2f27f05e86326f27134e79370f431c2c098348d/libcontainer/seccomp/config.go#L74
> 
> 
> https://github.com/opencontainers/runc/blob/a2f27f05e86326f27134e79370f431c2c098348d/libcontainer/seccomp/patchbpf/enosys_linux.go#L82

I have updated it, please check